ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -38,28 +41,28 @@ import           Ouroboros.Network.Protocol.Handshake.Version
 import           Ouroboros.Network.ErrorPolicy
 import           Ouroboros.Network.IOManager
 import           Ouroboros.Network.Mux
-import           Ouroboros.Network.NodeToClient ( NodeToClientVersion (..)
-                                                , NodeToClientVersionData )
+import           Ouroboros.Network.NodeToClient (NodeToClientVersion (..),
+                     NodeToClientVersionData)
 import qualified Ouroboros.Network.NodeToClient as NodeToClient
-import           Ouroboros.Network.NodeToNode ( NodeToNodeVersion (..)
-                                              , NodeToNodeVersionData
-                                              , AcceptedConnectionsLimit (..)
-                                              , AcceptConnectionsPolicyTrace (..)
-                                              , DiffusionMode (..)
-                                              , RemoteAddress
-                                              )
-import qualified Ouroboros.Network.NodeToNode   as NodeToNode
-import           Ouroboros.Network.Socket ( ConnectionId (..)
-                                          , NetworkMutableState
-                                          , newNetworkMutableState
-                                          , cleanNetworkMutableState
-                                          , NetworkServerTracers (..)
-                                          )
-import           Ouroboros.Network.Subscription.Ip
+import           Ouroboros.Network.NodeToNode
+                     (AcceptConnectionsPolicyTrace (..),
+                     AcceptedConnectionsLimit (..), DiffusionMode (..),
+                     NodeToNodeVersion (..), NodeToNodeVersionData,
+                     RemoteAddress)
+import qualified Ouroboros.Network.NodeToNode as NodeToNode
+import           Ouroboros.Network.Socket (ConnectionId (..),
+                     NetworkMutableState, NetworkServerTracers (..),
+                     cleanNetworkMutableState, newNetworkMutableState)
```

Comment:
Please preserve the original import style, and make sure the diff is minimal.  This will ensure that updating other branches when this is merged will be easy.